primefaces/src/main/java/org/primefaces/component/charts/ChartRenderer.java

```diff
@@ -164,7 +164,15 @@ protected void encodeScales(FacesContext context, String chartName, Object scale
             if (scales instanceof CartesianScales) {
                 writer.write("\"scales\":{");
                 CartesianScales cScales = (CartesianScales) scales;
-                StringBuilder scaleAttrs = new StringBuilder(20);
+                StringBuilder scaleAttrs = new StringBuilder(128);
+
+                if (cScales.getSuggestedMin() != null) {
+                    writeJsonAttribute(scaleAttrs, "suggestedMax", cScales.getSuggestedMin().toString());
+                }
+                if (cScales.getSuggestedMax() != null) {
+                    writeJsonAttribute(scaleAttrs, "suggestedMax", cScales.getSuggestedMax().toString());
+                }
+
```

Comment:
suggestedMin and -Max are per axis options, therefore they have to be encoded for each axis individually, i.e. https://www.chartjs.org/docs/latest/axes/#axis-range-settings